extensions/src/test/java/org/pentaho/platform/web/http/filters/PentahoWebContextFilterTest.java

```diff
@@ -162,54 +162,6 @@ public void testDoGetWithNoOsgiRequireConfigDefault() throws ServletException, I
     assertTrue( this.requirejsManagerInitIsCalled( response, null ) );
   }
 
-  @Test
-  public void testDoGetWithFullyQualifiedUrlTrue() throws ServletException, IOException {
```

Comment:
Recover the first two tests, that deal with the `fullyQualifiedUrl` parameter.

You can also modify the other two to tests to verify it uses the relative path by default.